packages/TelegramClient-UI.package/TCUAddMembersMessage.class/properties.json

```diff
@@ -0,0 +1,14 @@
+{
+	"category" : "TelegramClient-UI",
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "rgw 7/21/2022 15:08",
+	"instvars" : [
+		 ],
+	"name" : "TCUAddMembersMessage",
+	"pools" : [
+		 ],
+	"super" : "TCUMessage",
```

Comment:
We're not sure about that. TCUAddMembersMessage inherits from TCUMessage and TCCAddMembersMessage from TCCUserMessage. We lose the parallelism between core and UI with this :( This decision is understandable because an info message and a user message look the same in the core but not in the UI.

Reply:
After merging these PRs, we should remove the separation between TCCUserMessage and TCCMessage. We do lose parallelism, but TCUUserMessage has features we don't need